playbook-website/app/javascript/components/GlobalPropsAndTokens/ExamplesPage/Examples/LineHeightGlobalProps.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Flex, Card, Body } from "playbook-ui";
+import ShowPage from "../../Templates/ShowPage";
+import PropsExamplesTable from "../../Templates/Subcomponents/PropsExamplesTable";
+import ExampleCodeCard from "../../Templates/Subcomponents/ExampleCodeCard";
+
+const LineHeightGlobalProps = () => {
+
+    const VisualGuideExample = ({ text, lineHeight }: any) => {
+        return (
+      <>
+        <Flex orientation="column" alignItems="center" gap="xs">
+            <Card
+              className="visual_guide_card_line_height"
+              display="flex"
+              alignItems="center"
+              justifyContent="center"    
+            >
+              <Body text={text} lineHeight={lineHeight} />
+            </Card>
+            <Body text={lineHeight}  />
+        </Flex>
+      </>
+      )
+     }
+
+    const LineHeightCard = () => {
+        return (
+          <Flex gap="sm" wrap>
+            <VisualGuideExample text="The quick brown fox jumps over 17 lazy dogs." lineHeight="tightest" />
+            <VisualGuideExample text="The quick brown fox jumps over 17 lazy dogs." lineHeight="tighter" />
+            <VisualGuideExample text="The quick brown fox jumps over 17 lazy dogs." lineHeight="tight" />
+            <VisualGuideExample text="The quick brown fox jumps over 17 lazy dogs." lineHeight="normal" />
+            <VisualGuideExample text="The quick brown fox jumps over 17 lazy dogs." lineHeight="loose" />
+            <VisualGuideExample text="The quick brown fox jumps over 17 lazy dogs." lineHeight="looser" />
+            <VisualGuideExample text="The quick brown fox jumps over 17 lazy dogs." lineHeight="loosest" />
+          </Flex>
+        );
+    };
+
+
+    const values = [
+      "tightest",
+      "tighter",
+      "tight",
+      "normal",
+      "loose",
+      "looser",
+      "loosest",
+    ];
+    
+
+  return (
+    <>
+      <ShowPage
+        pageType="tokens"
```

Comment:
This also needs to say 'global_props' (global_props is the default so just removing this should be enough)